Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -42,6 +42,43 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
                 .to(throwError(ServerError.commandError(code: 59, message: "", errorLabels: nil)))
     }
 
+    func testDropDatabase() throws {
+        let center = NotificationCenter.default
+
+        let client = try MongoClient(options: ClientOptions(eventMonitoring: true))
+        client.enableMonitoring(forEvents: .commandMonitoring)
+
+        var db = client.db(type(of: self).testDatabase)
+        var writeConcern = try WriteConcern(journal: true, w: .number(1))
+
+        let collection = db.collection("collection")
+        try collection.insertOne(["test": "blahblah"])
+
+        var expectedWriteConcerns: [Document] = [
+                                                    ["w": Int32(1), "j": true],
+                                                    ["w": Int32(1), "j": true, "wtimeout": Int32(123)]
+                                                ]
+        let observer = center.addObserver(forName: nil, object: nil, queue: nil) { notif in
+            guard let event = notif.userInfo?["event"] as? CommandStartedEvent else {
+                return
+            }
+
+            expect(event.command["dropDatabase"]).toNot(beNil())
+            expect(event.command["writeConcern"]).toNot(beNil())
+            expect(event.command["writeConcern"] as? Document).to(equal(expectedWriteConcerns.removeFirst()))
```

Comment:
`removeFirst` crashes if the array you call it on is non-empty, so to be safe we might want to first check that `expectedWriteConcerns.isEmpty` (similar to [here](https://github.com/mongodb/mongo-swift-driver/blob/711a99ea5786ddcf056688b3100bffd457fd76bf/Tests/MongoSwiftTests/CommandMonitoringTests.swift#L62))